docs/getting-started.md

```diff
@@ -0,0 +1,138 @@
+## Getting Started
+
+### Prerequisites
+The following dependencies are needed to compile/build/run.
+* gcc/g++ >=4.9 ( std=c++14 is used )
+* meson >= 0.50
+* libopenblas-dev and base
+* tensorflow-lite >=1.14.0
+* libiniparser
+* libjsoncpp >=0.6.0 ( if you wand to use open AI )
+* libcurl3 >=7.47 ( if you wand to use open AI )
+* libgtest ( for testing )
```

Comment:
Provide the instruction to install the pre-requisites as - `sudo apt install ......` as well?
```

Reply:
I think we are not only support ubuntu. If we start to write this, we have to give them how-to install for all platform.